src/Morphir/Spark/AST.elm

```diff
@@ -309,6 +313,22 @@ expressionFromValue ir morphirValue =
                     expressionFromValue ir arg
                         |> Result.map (\expr -> Function "isnull" [ expr ])
 
+                Value.Apply _ (Value.Apply _ (Value.Reference _ ( [ [ "morphir" ], [ "s", "d", "k" ] ], [ [ "list" ] ], [ "member" ] )) item) items ->
+                    case items of
+                        Value.List _ list ->
```

Comment:
it's possible to match `Value.List _ list` directly on line 316.
It would save you the extract branch on line 330

Reply:
Thanks for that, I've pushed the branch with the case statement removed.